.github/workflows/linter.yml

```diff
@@ -0,0 +1,22 @@
+name: Go Linter
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          stable: 'false'
+          go-version: '1.16.1'
+
+      - name: Lint
```

Comment:
Thank you for opening a PR. I believe these checks are available in flyteorg/flytetools repo and should be referenced here as reusable workflows...
@evalsocket  can you send links for examples from other repos?

Reply:
@TheYk98 @EngHabu We need to change lint workflow here https://github.com/flyteorg/flytetools/blob/master/.github/workflows/lint.yml

Currently we didn't migrated flyteidl changes, @TheYk98 it would be great if you can use re use able workflow here like https://github.com/flyteorg/flytectl/blob/master/.github/workflows/checks.yml#L19

Reply:
changed wrt comment @evalsocket @EngHabu